integreat_cms/core/settings.py

```diff
@@ -1310,3 +1310,8 @@
 
 #: Zammad ticket group used for Integreat chat messages
 USER_CHAT_TICKET_GROUP: Final[str] = "integreat-chat"
+
+#: Duration (in hours) that we retain pending push notifications for retry attempts before discarding them
+NOTIFICATION_RETAIN_TIME_IN_HOURS: Final[int] = int(
+    os.environ.get("INTEGREAT_CMS_NOTIFICATION_RETAIN_TIME_IN_HOURS", 24)
+)
```

Comment:
Could you prefix this setting and add it to the firebase push notification chapter? For example below `FCM_SCHEDULE_INTERVAL_MINUTES`